style.css

```diff
@@ -0,0 +1,91 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+#nav-bar{
+    width: 100%;
+    position:fixed;
+    padding: 2rem 10%;
+    text-align: center;
+    margin-right: 2rem;
+    background-color: crimson;
+    color: burlywood;
+}
+
+a{
+    text-decoration: none;
+}
+ 
+#nav-bar a{
+    color: burlywood;
+    text-decoration: none;
+    font-weight: 400;
+    font-size: 18px;
+    margin-right: 2rem;
+}
+
+#welcome-section{
+    height: 100vh;
+    width: 100%;
+    background-color:beige;
+}
+
+#welcome-section h1{
+    font-size: 40px;
+    font-weight: 700;
+    color: crimson;
+    text-align: center;
+    padding: 10rem 10%;
+}
+
+.projects-title{
+    font-weight: 500;
+    font-size: 25px;
+    text-align: center;
+    color: crimson;
```

Comment:
I think it will be better to add a background image to your header it makes your site look much better and more professional